Tests/MapboxMapsTests/Style/StyleIntegrationTests.swift

```diff
@@ -129,4 +129,23 @@ internal class StyleIntegrationTests: MapViewIntegrationTestCase {
 
         wait(for: [expectation], timeout: 5.0)
     }
+
+    func testGetLocaleValue() {
+        guard let style = style else {
+            XCTFail("There should be valid MapView and Style objects created by setUp.")
+            return
+        }
+
+        // Case: base
+        var locale = Locale(identifier: "es")
+        var localeValue = style.getLocaleValue(locale: locale)
+
+        XCTAssertEqual(localeValue, "es")
+
+        // Case: when script and region codes does not exist
+        locale = Locale(identifier: "en-US")
+        localeValue = style.getLocaleValue(locale: locale)
+
+        XCTAssertEqual(localeValue, "en")
+    }
```

Comment:
other cases to check:
* handling of zh-Hans
* handling of zh-Hant
* handling of the special cases for streets v7 and streets v8

Reply:
* handling of unsupported locales

Reply:
Would still love to see a test for `zh-Hans`